search-api/search-service/conf/application.conf

```diff
@@ -293,7 +293,7 @@ search.config.path=/home/learning/platform/search
 cache.type="redis"
 
 search.es_conn_info="localhost:9200"
-search.fields.query=["name^100","title^100","lemma^100","code^100","domain","subject","description^10","keywords^100","ageGroup^10","filter^10","theme^10","genre^10","objects^25","contentType^100","language^200","teachingMode^25","skills^10","learningObjective^10","curriculum^100","gradeLevel^100","developer^100","attributions^10","identifier^100","IL_UNIQUE_ID^100","owner^50","board^100","relatedBoards^100","creator^100", "dialcodes^100","text","words","releaseNotes"]
+search.fields.query=["name^100.0","keywords^60.0","purpose^60", "source^50.0","description^5.0","language^5.0"]
```

Comment:
this should not be part of source code change... doing the change here affects default use cases for Sunbird.
This change should be added in deployment script for iGOT specific repo.